script/tool/lib/src/publish_check_command.dart

```diff
@@ -18,7 +18,17 @@ class PublishCheckCommand extends PluginCommand {
     Directory packagesDir,
     FileSystem fileSystem, {
     ProcessRunner processRunner = const ProcessRunner(),
-  }) : super(packagesDir, fileSystem, processRunner: processRunner);
+  }) : super(packagesDir, fileSystem, processRunner: processRunner) {
+    argParser.addFlag(
+      _allowPrereleaseFlag,
+      help: 'Allows the pre-release SDK warning to pass.'
+          'When enabled, a pub warning, which asks to publish the package as a pre-release version when'
```

Comment:
You're missing spaces at the ends of these string literals, so they won't concatenate correctly.

Reply:
The second part still has the same problem; this will print "[...] a pre-release verison whenthe SDK [...]"